docs/articles_en/openvino-workflow/model-optimization-guide/compressing-models-during-training/quantization-aware-training-tensorflow.rst

```diff
@@ -0,0 +1,131 @@
+      :fragment: [imports]
+
+2. Create NNCF configuration
+####################################
+
+Here, you should define NNCF configuration which consists of model-related parameters (``"input_info"`` section) and parameters
+of optimization methods (``"compression"`` section). For faster convergence, it is also recommended to register a dataset object
+specific to the DL framework. It will be used at the model creation step to initialize quantization parameters.
+
+.. tab:: TensorFlow
+
+   .. doxygensnippet:: docs/optimization_guide/nncf/code/qat_tf.py
+      :language: python
+      :fragment: [nncf_congig]
+
+
+3. Apply optimization methods
+####################################
+
+In the next step, you need to wrap the original model object with the ``create_compressed_model()`` API using the configuration
+defined in the previous step. This method returns a so-called compression controller and a wrapped model that can be used the
+same way as the original model. It is worth noting that optimization methods are applied at this step so that the model
+undergoes a set of corresponding transformations and can contain additional operations required for the optimization. In
+the case of QAT, the compression controller object is used for model export and, optionally, in distributed training as it
+will be shown below.
+
+.. tab:: TensorFlow
+
+   .. doxygensnippet:: docs/optimization_guide/nncf/code/qat_tf.py
+      :language: python
+      :fragment: [wrap_model]
+
+
+4. Fine-tune the model
+####################################
+
+This step assumes that you will apply fine-tuning to the model the same way as it is done for the baseline model. In the
+case of QAT, it is required to train the model for a few epochs with a small learning rate, for example, 10e-5. In principle,
+you can skip this step which means that the post-training optimization will be applied to the model.
+
+.. tab:: TensorFlow
+
+   .. doxygensnippet:: docs/optimization_guide/nncf/code/qat_tf.py
+      :language: python
+      :fragment: [tune_model]
+
+
+5. Multi-GPU distributed training
+####################################
+In the case of distributed multi-GPU training (not DataParallel), you should call ``compression_ctrl.distributed()`` before
+the fine-tuning that will inform optimization methods to do some adjustments to function in the distributed mode.
+
+.. tab:: TensorFlow
+
+   .. doxygensnippet:: docs/optimization_guide/nncf/code/qat_tf.py
+      :language: python
+      :fragment: [distributed]
+
+
+.. note::
+   The precision of weights gets INT8 only after the step of model conversion to OpenVINO Intermediate Representation.
+   You can expect the model footprint reduction only for that format.
+
+
+These were the basic steps to applying the QAT method from the NNCF. However, it is required in some cases to save/load model
+checkpoints during the training. Since NNCF wraps the original model with its own object it provides an API for these needs.
```

Comment:
```suggestion
These steps outline the basics of applying the QAT method from the NNCF. However, in some cases, it is required to save/load model
checkpoints during training. Since NNCF wraps the original model with its own object, it provides an API for these needs.
```